packages/location-icon/src/LocationIcon.story.tsx

```diff
@@ -1,4 +1,4 @@
-import React from "react";
```

Comment:
This story is not being shown in Storybook. Add the `.story.tsx` extension to `/.storybook/config.js`.